### PR TITLE
[#170048540] refactor docker-compose

### PR DESCRIPTION
**Pull request by gunzip**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please include a description of your PR changes.</td>
    </tr>
  </tbody>
</table>



## Affected stories

  * ⚙️ [#170048540](https://www.pivotaltracker.com/story/show/170048540): Rifattorizzare docker-compose

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 59e248aa95cb8f171fd0afa7c0703399395b32ed
</p>

File: .env.example [hunk 1]
```diff
@@ -1,7 +1,10 @@
-POSTGRES_HOST=database
-POSTGRES_DB=db
-POSTGRES_USER=io-onboarding-pa-api
-POSTGRES_PASSWORD=password
+NODE_ENV=development
+COMPOSE_PROJECT_NAME=io-onboarding
```
Comment (alexgpeppe) [inline .env.example]: This new env var seems to be unused, what is it needed for?

**Comment by alexgpeppe** [comment]
@gunzip, please also update the env variables in the README file accordingly.

File: .env.example (hunk 1 above)
Comment (gunzip) [reply .env.example]: https://docs.docker.com/compose/reference/envvars/

**Pull request merged**